docs/docs/configuration/generic-mapping.md

```diff
@@ -17,12 +20,17 @@ public static partial class ModelMapper
     public static partial TTarget Map<TTarget>(object source);
     // highlight-end
 
+    // highlight-start
+    public static partial TTarget MapFruit<TTarget>(Fruit source);
+    // highlight-end
+
```

Comment:
I don't think this documentation changes add any additional value. I'd either remove the old sample and keep the new one or only keep the old one.